<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>536</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>825</width>
    <height>536</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>.AppleSystemUIFont</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Map tool</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Desktop/Hellion Edited/PNG/Source File_Hellion-03.png</normaloff>../Desktop/Hellion Edited/PNG/Source File_Hellion-03.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="LicenseBarTop">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
     </font>
    </property>
    <property name="text">
     <string>License plate:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Search maps</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>287</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Create new</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>400</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
     </font>
    </property>
    <property name="text">
     <string notr="true">ECU</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>400</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
     </font>
    </property>
    <property name="text">
     <string notr="true">TCU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>430</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>370</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
     </font>
    </property>
    <property name="text">
     <string>Brand:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LicenseBarBot">
    <property name="geometry">
     <rect>
      <x>177</x>
      <y>370</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
     </font>
    </property>
    <property name="text">
     <string>License plate:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="brandName">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentText">
     <string notr="true">Audi</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <item>
     <property name="text">
      <string>Audi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bmw</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cadillac</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Citroen</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fiat</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ford</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Honda</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mazda</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mercedes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MINI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nissan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Opel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Peugeot</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Porsche</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Renault</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Saab</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Seat</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Skoda</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Subaru</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Toyota</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Volvo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VW</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="BrandNameBarTop">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
     </font>
    </property>
    <property name="text">
     <string>Brand:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="BrandNameTop">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentText">
     <string notr="true">Audi</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <item>
     <property name="text">
      <string>Audi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bmw</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cadillac</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Citroen</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fiat</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ford</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Honda</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mazda</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mercedes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MINI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nissan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Opel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Peugeot</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Porsche</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Renault</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Saab</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Seat</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Skoda</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Subaru</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Toyota</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Volvo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VW</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Papyrus</family>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>10</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <bold>false</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="pixmap">
     <pixmap resource="Hellion.qrc">:/new/prefix1/HellionLogoWhite.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundImg">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>-30</y>
      <width>711</width>
      <height>551</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>5</width>
      <height>5</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="midLineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Hellion.qrc">:/new/prefix1/car4.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTreeView" name="folderView">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>351</width>
      <height>471</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="allColumnsShowFocus">
     <bool>true</bool>
    </property>
    <attribute name="headerVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="headerStretchLastSection">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QToolButton" name="toggleTree">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>16</width>
      <height>131</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset theme="folder">
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonIconOnly</enum>
    </property>
    <property name="arrowType">
     <enum>Qt::RightArrow</enum>
    </property>
   </widget>
   <widget class="QLabel" name="sideBarBg">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>171</width>
      <height>511</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="pixmap">
     <pixmap resource="Hellion.qrc">:/new/prefix1/NewPStest.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="tuningGuideButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>103</y>
      <width>140</width>
      <height>49</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="resellerButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>140</width>
      <height>49</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="guideText">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>110</y>
      <width>95</width>
      <height>39</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Hellion.qrc">:/new/prefix1/TuningText.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="resellerText">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>161</y>
      <width>66</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Hellion.qrc">:/new/prefix1/ResellerText.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="GuideIcon">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>115</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Hellion.qrc">:/new/prefix1/TuningGuide.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="resellerIcon">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>165</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Hellion.qrc">:/new/prefix1/Reseller2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Hellion.qrc">:/new/prefix1/MapToolLogo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>-10</y>
      <width>20</width>
      <height>521</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <zorder>backgroundImg</zorder>
   <zorder>sideBarBg</zorder>
   <zorder>LicenseBarTop</zorder>
   <zorder>lineEdit</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>pushButton</zorder>
   <zorder>checkBox</zorder>
   <zorder>checkBox_2</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>label_4</zorder>
   <zorder>LicenseBarBot</zorder>
   <zorder>brandName</zorder>
   <zorder>BrandNameBarTop</zorder>
   <zorder>BrandNameTop</zorder>
   <zorder>line</zorder>
   <zorder>label_7</zorder>
   <zorder>folderView</zorder>
   <zorder>toggleTree</zorder>
   <zorder>guideText</zorder>
   <zorder>resellerText</zorder>
   <zorder>GuideIcon</zorder>
   <zorder>resellerIcon</zorder>
   <zorder>label_11</zorder>
   <zorder>tuningGuideButton</zorder>
   <zorder>resellerButton</zorder>
   <zorder>line_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>455</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionEnglish"/>
     <addaction name="actionSwedish"/>
    </widget>
    <addaction name="actionfilePath"/>
    <addaction name="separator"/>
    <addaction name="menuLanguage"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionInstructions"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionfilePath">
   <property name="text">
    <string>File path</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionInstructions">
   <property name="text">
    <string>Instructions</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionSwedish">
   <property name="text">
    <string>Swedish</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Hellion.qrc"/>
 </resources>
 <connections/>
</ui>
